src/sssom/writers.py

```diff
@@ -271,29 +290,78 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
 
-    TODOs
+    TODO: add to CLI & to these functions: r4 vs r5 param
+    TODO: What if the msdf doesn't have everything we need? (i) metadata, e.g. yml, (ii) what if we need to override?
+     - todo: later: allow any nested arbitrary override: (get in kwargs, else metadata.get(key, None))
+
+    Minor todos
+    todo: mapping_justification: consider `ValueString` -> `ValueCoding` https://github.com/timsbiomed/issues/issues/152
     todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
-    TODO: Add additional fields from both specs
-     - ConceptMap spec fields: https://www.hl7.org/fhir/r4/conceptmap.html
-      - Joe: Can also utilize: /Users/joeflack4/projects/hapi-fhir-jpaserver-starter/_archive/issues/sssom/example_json/minimal.json
-     - SSSOM more fields:
-     - prefix_map
-     - SSSOM spec fields: https://mapping-commons.github.io/sssom/Mapping/
     """
+    # Constants
     df: pd.DataFrame = msdf.df
+    # TODO: R4 (try this first)
+    #  relatedto | equivalent | equal | wider | subsumes | narrower | specializes | inexact | unmatched | disjoint
+    #  https://www.hl7.org/fhir/r4/conceptmap.html
+    # todo: r4: if not found, should likely be `null` or something. check docs to see if nullable, else ask on Zulip
+    # TODO: R5 Needs to be one of:
+    #  related-to | equivalent | source-is-narrower-than-target | source-is-broader-than-target | not-related-to
+    #  https://www.hl7.org/fhir/r4/valueset-concept-map-equivalence.html
+    #  ill update that next time. i can map SSSOM SKOS/etc mappings to FHIR ones
+    #  and then add the original SSSOM mapping CURIE fields somewhere else
+    # https://www.hl7.org/fhir/valueset-concept-map-equivalence.html
+    # https://github.com/mapping-commons/sssom-py/issues/258
+    equivalence_map = {
+        # relateedto: The concepts are related to each other, and have at least some overlap in meaning, but the exact
+        #   relationship is not known.
+        "skos:related": "relatedto",
+        "skos:relatedMatch": "relatedto",  # canonical
+        # equivalent: The definitions of the concepts mean the same thing (including when structural implications of
+        #   meaning are considered) (i.e. extensionally identical).
+        "skos:exactMatch": "equivalent",
+        # equal: The definitions of the concepts are exactly the same (i.e. only grammatical differences) and structural
+        #   implications of meaning are identical or irrelevant (i.e. intentionally identical).
+        "equal": "equal",  # todo what's difference between this and above? which to use?
+        # wider: The target mapping is wider in meaning than the source concept.
+        "skos:broader": "wider",
+        "skos:broadMatch": "wider",  # canonical
+        # subsumes: The target mapping subsumes the meaning of the source concept (e.g. the source is-a target).
+        "rdfs:subClassOf": "subsumes",
+        "owl:subClassOf": "subsumes",
```

Comment:
This `owl:subClassOf` does not exist, just FYI.

Reply:
Thanks for catching! Not sure how that got in there. Removed.